alsonick.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Nicholas Njoki",
+  "type": "personal",
+  "bio": "I'm a full-stack developer using tools like TypeScript & React.",
+  "avatar": "https://notnick.io/pfp.png",
```

Comment:
Hi @alsonick As per the recent updates, No need to add the avatar
You can remove it. Thank you

Reply:
Thanks for the reply, I've now removed the avatar property. 🙂